CMakeLists.txt

```diff
@@ -411,9 +459,6 @@ if(HAIKU)
 endif()
 
 if(NOT CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
-  # Change __FILE__ to only show the path relative to the project folder
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-builtin-macro-redefined -D'__FILE__=\"$(subst $(realpath ${CMAKE_SOURCE_DIR})/,,$(abspath $<))\"'")

```

Comment:
Find a way to do this with out breaking ninja, see also https://github.com/diasurgical/devilutionX/issues/490

Reply:
This was just fixed by @sweetgiorni :)